packages/wdio-browserstack-service/src/service.ts

```diff
@@ -235,4 +236,13 @@ export default class BrowserstackService implements Services.ServiceInstance {
             log.info(`${browserString} session: ${response.body.automation_session.browser_url}`)
         })
     }
+
+    _parentSuiteName(testDescription: string, otherTestDescription: string): string {
```

Comment:
Can we name these properties a bit more meaningful?

Reply:
Do you have any suggestions?

These are two Jasmine test descriptions and the method determines which prefix (think: "describes") they have in common.